webpify.js

```
@@ -0,0 +1,184 @@
+                            content = content.replace(new RegExp(absolutePath, 'g'), absoluteWebPPath);
+                            hasChanges = true;
+                            console.log(`Replaced absolute path reference in ${filePath}: ${absolutePath} → ${absoluteWebPPath}`);
+                        }
+                    }
+                }
+
+                // Original relative path handling
+                const absoluteOriginalPath = path.resolve(original);
+                const regex = new RegExp(`(["'\\(])([^"']*${fileName.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&')})\\1`, 'g');
+                let match;
+                while ((match = regex.exec(content)) !== null) {
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)